### PR TITLE
accept CVRs with an array of page numbers, not just a single page number

### PR DESCRIPTION
**Pull request by benadida**

### CODE REVIEW

File: client/src/lib/votecounting.test.ts [hunk 1]
```diff
@@ -215,6 +215,49 @@ test('parsing CVRs flags when page number is set but not a number', () => {
...
 })
 
+test('parsing CVRs flags when page numbers is set but not an array of numbers', () => {
+  const cvr: CastVoteRecord = {
+    _ballotStyleId: '12',
+    _precinctId: '23',
+    _ballotId: 'abc',
+    _scannerId: 'scanner-1',
+    _testBallot: false,
+    // @ts-ignore
```
Comment (eventualbuddha) [inline client/src/lib/votecounting.test.ts:225]: `@ts-expect-error` is better here is the version of TS is recent enough to support it.

File: client/src/lib/votecounting.ts [hunk 2]
```diff
@@ -550,6 +570,7 @@ const processCastVoteRecord = ({
     _testBallot: castVoteRecord._testBallot,
     _scannerId: castVoteRecord._scannerId,
     _pageNumber: castVoteRecord._pageNumber,
+    _pageNumbers: castVoteRecord._pageNumbers,
```
Comment (eventualbuddha) [inline client/src/lib/votecounting.ts:573]: Should we be normalizing them into one field? Or keep both?

File: client/src/lib/votecounting.test.ts [hunk 3]
```diff
@@ -215,6 +215,49 @@ test('parsing CVRs flags when page number is set but not a number', () => {
...
+    {
+      cvr,
+      errors: [
+        'Page number in CVR must be either _pageNumber, or _pageNumbers, but cannot be both.',
+      ],
+      lineNumber: 1,
+    },
+  ])
+})
+
```
Comment (eventualbuddha) [inline client/src/lib/votecounting.test.ts:260]: Is there a happy path test?

File: client/src/lib/votecounting.ts (hunk 2 above)
Comment (benadida) [reply client/src/lib/votecounting.ts:573]: I think we keep both, good to capture that it is old vs new format.

File: client/src/lib/votecounting.test.ts (hunk 1 above)
Comment (benadida) [reply client/src/lib/votecounting.test.ts:225]: this is not working for me.

**Pull request merged**